products/pluto/pluto_build/sql/qaqc_bbl_diffs.sql

```diff
@@ -0,0 +1,30 @@
+DROP TABLE IF EXISTS qaqc_bbl_diffs;
+CREATE TABLE qaqc_bbl_diffs (
+    source_version text,
+    bbl bigint,
```

Comment:
It looks like we have `bbl` in a few other places. I'm curious why... is it just to remove leading zeros? cc @fvankrieken @damonmcc

Reply:
could be, or part of a battle against floats. I think it's an inherited choice rather than one that we made lol

seems like BBLs should be considered strings to me since they're IDs and we'd never want to treat them like numbers

Reply:
@damonmcc, I agree. I will update `bbl` type the `qaqc_bbl_diffs` table. However, my understanding is that we will revise `bbl` type to be of text type in all other generated tables during pluto build separately, not in this PR. Lmk if that's correct.